src/web/routes.rs

```diff
@@ -85,7 +85,7 @@ pub(super) fn build_routes() -> Routes {
     );
     routes.internal_page(
         "/crate/:name/:version/builds/:id",
-        super::builds::build_list_handler,
+        super::build_details::build_details_handler,
```

Comment:
Yes, it's one build per page. It no longer shows the list of other builds and limits below the build log, which means it doesn't need the additional code for loading that data in this handler too. You can just hit back to go back to the list.

It doesn't drastically simplify the code right now, but I think it will help a lot once the `build_details_handler` needs to start querying the list of different targets to show.

Reply:
Hmm, my concern was that people would no longer have the limits staring them in the face when they see a failed build, but it looks like the 'build failed' link takes you to /builds directly, so you do see the limit table before seeing the log itself. I'm still slightly worried people won't think of it, but it's not a big deal, I'd be ok either way.